File: tests/Fortran/Fixed/0027-hostdata.f
      program main
      integer :: a(1000),b(1000)
      integer :: i
      !$acc data copyin(a,b)
        !$acc kernels
        do i = 1, 1000
          a(i) = i
        enddo
        !$acc end kernels
        !$acc host_data use_device(a,b)
        !$acc end host_data
      !$acc end data
      end program

File: tests/C/0027-hostdata.c
int main()
{
	int a[1000],b[1000];
	#pragma acc data copyin(a,b)
	{
		#pragma acc kernels
		{
			for (int i = 0; i < 1000; ++i)
				a[i] = i;
		}
		#pragma acc host_data use_device(a,b)
		{
		}
	}
	return 0;
}
